ksql-engine/src/test/java/io/confluent/ksql/structured/SelectValueMapperTest.java

```diff
@@ -56,9 +59,7 @@
   private final KsqlConfig ksqlConfig = new KsqlConfig(Collections.emptyMap());
 
   @Mock
-  private StructuredLogger processingLogger;
-
-  private ProcessingLogContext processingLogContext = ProcessingLogContext.create();
```

Comment:
nit: with these deletions, StructuredLogger and ProcessingLogContext can be removed as imports. Same comment for KsqlDelimitedDeserializerTest, KsqlJsonDeserializerTest, SerdeTestUtils, and KsqlGenericRowAvroDeserializerTest (after removing the unused `recordLogger`).